src/components/ComputeModal.js

```diff
@@ -415,6 +424,7 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
 
     return {
       hasGpu: computeConfig.hasGpu,
+      autopauseThreshold: computeConfig.autopauseThreshold,
```

Comment:
can this be a field in the runtime as opposed to a top-level field? Not sure what exactly the distinction is, but I view it as part of a runtime. Curious to hear your thoughts